packages/@o3r/schematics/package.json

```diff
@@ -16,6 +16,9 @@
     "prepare:build:builders": "yarn cpy 'schematics/**/*.json' dist/schematics && yarn cpy 'collection.json' dist",
     "build:builders": "tsc -b tsconfig.builders.json --pretty && yarn generate-cjs-manifest"
   },
+  "bin": {
+    "environment": "./dist/src/cli/environment.js"
```

Comment:
`@o3r/dev-tools` is a devDep of all packages and so of `@o3r/schematics`
I could move the helper into @o3r/dev-tools` but it means that it will become an optional peerDep of all packages
that in fact is mandatory if we want to run any schematics or builders.